docs/examples/hypothesis_property_based_test.py

```diff
@@ -0,0 +1,24 @@
+import typing
+import pydantic
```

Comment:
we normally do `from pydantic import BaseModel, PaymentCardNumber, PositiveFloat, EmailStr` unless there's a reason to required it this way, better to match the rest of the docs.